### DISCUSSION TITLE
Hi Everyone i m doing some due diligence [Aug 19]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6244] [Discord Threads]
> 🙋‍♂️ **arturvisualsitemaps**

**Comment by o1lab** [comment]
> 🙋‍♂️ **arturvisualsitemaps**  
#### looking at the repo, they are concerned with the `+500`  *current outstanding issues.*  
#  
> 🙋‍♂️ **morne04**  
#### To be fair, you need to exclude all issues marked as Feature requests and enhancements etc...  
#  
> 💁 **o1lab**  
#### <@649659153379360778> Some of the answers might be in why are you looking at alternatives ? Primary use here is user can connect to their own DB and also build airtable like bases. And  as Morne mentioned many of them are feature requests. Not to forget any open source repo would have 100s to 1000s of issues 🙂  
#  
> 🙋‍♂️ **nachteulen.coffee**  
#### We did check a lot before starting with NocoDB.  You get good stuff and it's open source. I'm ramping up a pilot now for nearly over a year and finding more use cases in which Nocodb supports us. at the end, is about your customer budget and what you can explain them.    
#### Are they up to pay your time to see how much can they cover with Nocodb? as a test. or they want to have a final answer?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1128745225691598938)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
